spack/packages/shumlib/package.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import shutil
+from spack.package import MakefilePackage
+
+
+class Shumlib(MakefilePackage):
+
+    """Shared UM Library.
+
+    Shumlib is the collective name for a set of libraries which are
+    used by the UM; the UK Met Office's Unified Model, that may be of
+    use to external tools or applications where identical
+    functionality is desired. The hope of the project is to enable
+    developers to quickly and easily access parts of the UM code that
+    are commonly duplicated elsewhere, at the same time benefiting
+    from any improvements or optimisations that might be made in
+    support of the UM itself.
+    """
+
+    homepage = "https://code.metoffice.gov.uk"
+    url = "https://github.com/metomi/shumlib/archive/refs/tags/um13.0.tar.gz"
```

Comment:
Hey @ScottWales , based on comments from @andrewcoughtrie I believe the `shumlib` version currently being used in `LFRic_apps` is version `um13.7` = `apps2.1` = `revision 7655`. Should we update to this version to be consistent? I don't think it is available on Github (only from MOSRS), so should we extract it from there?

Reply:
Hey @andrewcoughtrie , since we don't have a timeline on the Shumlib Github mirror, and the `um13.0` version is working for LFRic_apps at head of trunk, should we just merge this in (we need something so that we can rehabilitate our CI pipelines :) )

Reply:
I've added a new 'apps2.1' version to shumlib that pulls from the subversion repository. The default version is still '13.0'.

Reply:
For the time being I agree it might as well just use the SRS version, @yaswant is going to look into getting the latest versions into the metomi/shumlib GitHub repo.